doc/tntcxx_api.rst

```diff
@@ -28,141 +26,184 @@ Methods:
 
 .. _tntcxx_api_connector_connect:
 
-.. c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
+..  c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
 
-   Connects to a Tarantool instance that is listening on ``addr:port``.
+    Connects to a Tarantool instance. If a host doesn't reply within a timeout
+    period (2 seconds by default), an error is returned.
```

Comment:
How can I change the timeout? Should I?

Reply:
Until recently the timeout for this function wasn't editable. Now it's implemented as an optional argument. The description is corrected accordingly.